packages/navigator/src/browser/navigator-preferences.ts

```diff
@@ -34,13 +34,19 @@ Configure glob patterns for excluding files and folders from the navigator. A re
             default: {
                 '**/.git': true
             }
-        }
+        },
+        'navigator.singleClickPreview': {
+            type: 'boolean',
+            description: 'Single-click or selection of a file in the Navigator open the given file in preview mode. If no preview is available for the given file type, single click will open the file.',
```

Comment:
minor typo, I think you meant `opens` rather than `open`.
perhaps the description can be shortened (it will most likely be cut in the preferences widget tree) to something more easily read by end-users.

example: ` controls whether editors are open as previews when selected or single-clicked` ? 
we can debate however on what the description can ultimately be.